From a09b1f454d01799f4976cb1396348bd850e7223a Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 1 May 2017 14:59:57 -0400
Subject: [PATCH 12/22] makeguids: free our input buffer.

Covscan noticed this, but didn't notice that this is a short lived
program that just parses some tables during the build.  It *sooo*
doesn't matter.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/makeguids.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/makeguids.c b/src/makeguids.c
index da2f5f7..6a76c46 100644
--- a/src/makeguids.c
+++ b/src/makeguids.c
@@ -212,6 +212,7 @@ main(int argc, char *argv[])
 	close(in);
 	close(guidout);
 	close(nameout);
+	free(inbuf);
 
 	return 0;
 }
-- 
2.12.2

